_1_basics/src/main/java/code/_4_student_effort/FizzBuzz.java

```diff
@@ -0,0 +1,30 @@
+package code._4_student_effort;
+
+public class FizzBuzz {
+    public static void main(String[] args)
+    {
+        int i;
+        for(i=1; i<=100; i++)
+        {
+
+
+                if (i % 3 == 0)
+                {
+                    System.out.println("Fizz");
+                    i++;
+                    if (i % 5 == 0)
+                    {
+                        System.out.println("Buzz");
+                        i++;
+                        if (i % 3 == 0 && i % 5 == 0)
+                        {
+                            System.out.println("FizzBuzz");
+                            i++;
+                        }
+                    }
+                }
+                else
+                    System.out.println(i);
```

Comment:
implementarea propusa este OK, insa as vrea sa fii constienta de implicatiile pe care le aduce ... si anume un efort suplimentar in "maintainability" ... daca ar fi sa implementezi si stage 2, unde adaugi Rizz si Jazz, atunci numarul de decizii luate in cod, creste exponential ...